docs/src/tutorial.md

```diff
@@ -6,10 +6,11 @@ Read through the official tutorial! Only the differences from the Python version
 
 ## Load MNIST data
 
-The `DataLoader` API provided in "examples/mnist_loader.jl" has some simple code for loading the MNIST dataset, based on the MNIST.jl package.
+The `DataLoader` API provided in `"examples/mnist_loader.jl"` has some simple code for loading the MNIST dataset, based on the `MNIST.jl` package.
 
 ```julia
-loader = data_loader()
+include(joinpath(Pkg.dir(), "TensorFlow", "examples", "mnist_loader.jl"))
```

Comment:
This can be written as
`include(Pkg.dir("TensorFlow", "examples", "mnist_loader.jl"))`
The `Pkg.dir` command has built in `joinpath`